pkg/networkmanager/interfaces.js

```diff
@@ -785,18 +824,20 @@ export function NetworkManagerModel() {
             connections_by_uuid[settings.connection.uuid] = obj;
     }
 
-    function refresh_settings(obj) {
+    async function refresh_settings(obj) {
```

Comment:
AFAICS this now does not change behaviour, it is just a rewrite of this function to be `async`, right? Please split this out into a separate commit.

Reply:
reverted.